src/app/components/drawer/base-drawer.tsx

```diff
@@ -103,13 +103,13 @@ export const BaseDrawer = memo((props: BaseDrawerProps) => {
         maxHeight={['calc(100vh - 24px)', 'calc(100vh - 96px)']}
       >
         <Box
-          css={css({
+          css={{
```

Comment:
cc @pete-watters I fixed this here bc scrolling was broken in the drawer after your PR.

Reply:
Thanks! Apologies for messing it up